### PR TITLE
Adds content for healthcare application landing page.

### PR DESCRIPTION
**Pull request by crwallace**
- This includes all content from @EmilyTav for the healthcare application landing page.

### CODE REVIEW

**Comment by emilyville** [comment]
@crwallace let me know when you guys are done kicking around the content. It would be good to chat about how we get people to this page. (I have a few different ideas)

**Comment by akainic** [comment]
@crwallace can you please include a screenshot of the page in the PR description?

File: _health-care/apply.md [hunk 1]
```diff
@@ -1,61 +1,74 @@
...
 <div class="primary" markdown="0">
 <div class="row" markdown="0">
-<div class="small-12 columns" markdown="1">
+<div class="small-12 columns usa-content" markdown="1">
+<div markdown="1">
+The VA operates America’s largest integrated health care system, with over 700 sites of care. It is consistently ranked among the nation’s top health care providers.
 
-### Who should read this
-Veterans seeking health care. 
+Health care at the VA includes preventive, primary, and specialty care; prescriptions; mental health care; home health care; geriatrics and extended care; medical equipment and prosthetics; and more. Most Veterans qualify for cost-free health care services, although some Veterans must pay modest copays for health care or prescriptions.
```
Comment (akainic) [inline _health-care/apply.md]: I know this was from the original content, but there shouldn't be semicolons separating the list items, just commas.

**Comment by akainic** [comment]
Looks like the new urls we're going to be using are:
landing page: healthcare/apply
application: healthcare/apply/application

**Comment by crwallace** [comment]
Hey Team (@akainic @emilyville @gnakm ) ! I would love to chat about the initial stab I took at this.

screencapture-localhost-4000-health-care-apply-1463936969445

We have a lot going on with the "Tips" so I would love some input. I played around with different colors but it wasn't working between accessibility and making sure that the colors did not give the wrong signal (for example: if the green button means apply I should not have a tip for people that don't qualify in green.)

**Comment by EmilyTav** [comment]
Looks great, thanks @crwallace 

A few initial thoughts--curious what others think, and we may want to just have a 30 min call to discuss for the sake of efficiency. 

- def a lot going on with the tips, but i do think they are pretty key. i wonder if maybe those should go in a box (like the quals currently are) and call the quals out with a bold/visual vertical align (i don't know what it's called, but a bold line that runs down the left side of indented/called out text

- the bold text that intros the reader to the page is a lot. maybe we make that standard text and start the page with "if you served on active duty, and separated under conditions other than dishonorable, you may qualify for VA healthcare."

- also, should we indent the standard copy slightly? that might just be me.

**Comment by crwallace** [comment]
@emilyville  and @awong-dev: This travis build is failing because I am referencing a route that doesn't exist in this repo `healthcare/apply/application`. Should I just reference the entire url `http://www.vets.gov/healthcare/apply/application`?

**Comment by emilyville** [comment]
Please loop me in on this guys, we'd like to make this page consistent with other assets. Just small stuff like we prefer not to say "frequently asked questions", and are trying to streamline our light blue box usage.

**Comment by gnakm** [comment]
@crwallace this looks great! I have some feedback:

1. Apply for VA Health Care buttons. Could we have the first button look like the one on this page with it on the top right in a gray bar. https://www.vets.gov/disability-benefits/conditions/ and take out the green bar in the bottom?

2. Italicized tips. This is super minor. I think because italics are not used throughout vets.gov, maybe it's better to avoid it. I've seen tips where it's just bolded:
 ex. **Tip**: This is an example of a tip

@emilyville Thoughts on these?

**Comment by webinista** [comment]
Making the _Apply for VA Health Care_ buttons look like the ones on the [Disability Benefits > Conditions](https://www.vets.gov/disability-benefits/conditions/ ) page would make this a variation of the `1-topic-landing` template.

I've opened #2397 to track that issue. Once that template is updated and merged, we can revisit this page and remove a lot of the markup.

**Comment by EmilyTav** [comment]
Oh one more comment on this: could we remove FAQs all together? so the sections are:

"am i eligible?" 
"how long will processing my application take?"

**Comment by emilyville** [comment]
screencapture-hca-apply

There's still a few tweaks that need to happen here and there (@webinista has a PR I want to incorporate https://github.com/department-of-veterans-affairs/vets-website/pull/2401 still so this is a 90% there version and there are some nits like link underlines.)

Also need to figure out what to do with the routes! 

This is the track changes doc with some changes:
[Need health care-MAB edits.docx](https://github.com/department-of-veterans-affairs/vets-website/files/280791/Need.health.care-MAB.edits.docx)

I made a few changes as I implemented this based on consistency with other pages that have not yet been incorporated into this doc, but I'm done for the evening so I will send an updated one tomorrow. 

PTAL @EmilyTav @staceylanger @crwallace

**Comment by emilyville** [comment]
Also don't merge this yet, it's a WIP.

**Comment by staceylanger** [comment]
I'm inclined to get rid of the blue box "Who should read this?" for a few reasons:
1. We mention the same content below.
2. The action we want is not "read this" it's "Apply"
3. From a visual priority perspective, it takes precedence over the apply button which isn't what we want.

**Comment by emilyville** [comment]
Awesome! incorporated. I'm going to stop posting screenshots, and instead send you all to this link:

https://vets-website-pr-2391.herokuapp.com/health-care/apply/

**Comment by EmilyTav** [comment]
- i like the lead text :us: 

- the two "apply" prompts are a bit distracting to me (might just be me, but if I already see it front and center w/the description that's where I'd be going--maybe something to test out!)

- the copy following "Who should read this?" doesn't seem to really follow the question. My inclination is to lead with that copy as the last sentence in the intro text

- we may need to remove the confirmation # line in the "How long will it take..." section. I got some feedback yesterday that makes me think the # is actually not searchable in ES yet, so changes to the backend need to occur to make that feasible. I have to confirm. 

- i'm tempted to have a "helpful documents" section (ie. stuff that will help them fill out the form quickly) for the tax return and insurance information

**Comment by EmilyTav** [comment]
Also, thank you all!

**Comment by emilyville** [comment]
@EmilyTav No problem :)
- content for a "documents you'll need" section would be great! this would help make it consistent with our other apply pages as well
- I'll take another pass once you know for sure about the confirmation #, etc
- Re: Who should need this... This is removed from the most recent version via Stacey's comment. Here's the latest:
https://vets-website-pr-2391.herokuapp.com/health-care/apply/

**Comment by akainic** [comment]
@crwallace @EmilyTav @emilyville Looks like there are some content changes to be made from the last conversation on this PR. Who is going to take those? Happy to it myself unless Courtney wants to jump back on this! We should also change the url for the page to be healthcare/apply.

File: Rakefile [hunk 2]
```diff
@@ -131,7 +131,7 @@ namespace :tests do
...
+    sh "bundle exec htmlproof ./_site --only-4xx --disable-external --check-favicon --check-html --allow-hash-href --href-ignore '/veterans-employment-center/','/gi-bill-comparison-tool/','/disability-benefits/learn/eligibility/.*','/disability-benefits/learn/','/disability-benefits/apply-for-benefits/','Employment-Resources/','/gibill/','/disability-benefits/','/employment/job-seekers/skills-translator','/employment/job-seekers/search-jobs','/employment/job-seekers/create-resume','/employment/commitments','/employment/users/sign_in','/employment/employers','/employment/employers/','/health-care/apply/application/'"
```
Comment (webinista) [inline Rakefile]: Should `/health-care/apply/application/` be `/healthcare/apply/application/`?

File: _sass/_hca.scss [hunk 3]
```diff
@@ -1,4 +1,14 @@
 // TODO: properly incorporate usds web standards for this
+.hca-landing-page {
```
Comment (webinista) [inline _sass/_hca.scss]: This block _shouldn't_ be necessary. Can you try removing it and see if the underline returns?

File: _sass/_hca.scss [hunk 4]
```diff
@@ -1,4 +1,14 @@
...
+  a.va-button-primary {
+    text-decoration: none !important;
+  }
+}
+
+#frequently-asked-questions {
```
Comment (webinista) [inline _sass/_hca.scss]: It looks like `#frequently-asked-question`  was removed from the HTML, so we can remove it from the CSS too.

**Pull request merged**